src/ShallowWrapper.js

```diff
@@ -321,7 +321,7 @@ export default class ShallowWrapper {
    * @returns {CheerioWrapper}
    */
   render() {
-    return cheerio.load(this.html()).root();
+    return this.type() === null ? cheerio() : cheerio.load(this.html()).root();
```

Comment:
yes, previously it threw `TypeError: Cannot read property 'parent' of null` from inside cheerio.

`cheerio()` and `cheerio(null)` and `cheerio('')` all return an empty cheerio wrapper - i just chose the simplest of these